release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -88,6 +88,22 @@ The xref:../support/remote_health_monitoring/using-insights-to-identify-issues-w
 [id="ocp-4-7-storage"]
 === Storage
 
+[id="ocp-4-5-persistent-storage-csi-ebs"]
+==== Persistent storage using the GCP PD CSI Driver Operator (Technology Preview)
+
+You can now use the Container Storage Interface (CSI) to deploy the CSI driver you need for provisioning Google Cloud Platform (GCP) persistent disks (PD). This Operator is in Technology Preview.
```

Comment:
// Will add the related xref when GCP PD CSI Driver Operator storage doc is approved and merged. (Tracked here: https://issues.redhat.com/browse/OSDOCS-1720.)

Reply:
This deployment should be done automatically on GCP clouds, so it's not something the user has to deploy. Maybe the following:

> The Google Cloud Platform (GCP) persistent disk CSI driver is automatically deployed and managed on GCP environments, allowing you to dynamically provision these volumes without the need for installing the driver manually. The Operator that manages this driver is in Technology Preview.